spec/forms/state_file/nj_household_rent_own_form_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe StateFile::NjHouseholdRentOwnForm do
+  let(:intake) { create :state_file_nj_intake }
+
+  describe "validations" do
+    let(:form) { described_class.new(intake, invalid_params) }
+
+    context "invalid params" do
+      context "all fields are required" do
+        let(:invalid_params) do
+          {
+            :household_rent_own => nil,
```

Comment:
[dust] this feels like a dumb question but isn't the value 0 when unfilled, not nil?

Reply:
same silly question below, when is the value `"rent"` vs when is it `1`?